mkdocs.template.yml

```diff
@@ -2,7 +2,7 @@ site_name: uv
 theme:
   name: material
   logo: assets/logo-letter.svg
-  favicon: assets/favicon.svg
+  favicon: assets/favicon.ico
```

Comment:
I guess we have to keep something here? I don't know if this will be used or the links in the override:

```
<link rel="icon" type="image/png" sizes="32x32" href="/static/favicon-32x32.png"/>
<link rel="icon" type="image/png" sizes="16x16" href="/static/favicon-16x16.png"/>
```

Reply:
It looks like this gets used.